.github/workflows/tiobe.yaml

```diff
@@ -0,0 +1,39 @@
+name: TICS scan
+
+on:
+  schedule:
+    - cron: "0 12 * * 3" # wednesday at 12:00
+
+jobs:
+  tiobe:
+    runs-on: ubuntu-24.04
```

Comment:
You might want to change this to `    runs-on: [self-hosted, linux, x64, "${{ (github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && inputs.tiobe == true)) && 'large-tiobe' || 'large' }}"]` to take advantage of our optimize runners.